00XX:reassert-fee-payment-method

```diff
@@ -0,0 +1,54 @@
+## Summary
+
+This HIP proposes changing the method of payment for Location Assert Fees from HNT tokens to IOT tokens. The primary purpose of this change is to drive value into IOT tokens and provide them with a small amount of utility. This HIP does not propose altering any other aspect of Location Assert Fees. The proposed changes are intended to be implemented in compliance with the rules set forth in [HIP-51](https://github.com/helium/HIP/blob/main/0051-helium-dao.md).
+
+## Motivation
+
+Currently, Location Assert Fees are paid using HNT tokens. However, there is no incentive to pay for onboarding using HNT under HIP-51, as none of the burned tokens accrue back to the subDAO. This proposal seeks to address this issue by transitioning the method of payment to IOT tokens.
+
+By adopting IOT tokens as the method of payment, we aim to drive value into the IOT ecosystem and give IOT tokens a small amount of utility. Moreover, this change aligns with the rules established in HIP-51, as detailed below.
+
+## Specification
+
+### Method of Payment
+
+This HIP proposes that Location Assert Fees be paid using IOT tokens instead of HNT tokens. The change will be effected by modifying the relevant code within the Helium network core implementation. This modification will ensure that only IOT tokens are accepted for Location Assert Fees.
+
+Data credits will not be utilized in this process. Instead, a straight burn of IOT tokens will be implemented, as using data credits across multiple subDAOs would circumvent the rules established in HIP-51.
+
+### Implementation
+
+A burn wallet will be used for the implementation of this change. The burn wallet will take the oracle value of IOT at the time of the transaction, ensuring that the correct amount of IOT tokens is burned for each Location Assert Fee transaction. The burn wallet will also ensure that no changes are made to any other aspect of Location Assert Fees.
+
+#### Burn Wallet Functionality on Solana
+
+A burn wallet on Solana functions by sending tokens to an address that has no known private key. This ensures that the tokens sent to this address are effectively removed from circulation and considered "burned". This process is irreversible.
+
+#### Flowchart of Burn Transactions
+
+1. User initiates a Location Assert Fee transaction.
+2. The transaction sends the required amount of IOT tokens, based on the current oracle value, to the burn wallet address.
+3. IOT tokens are burned, effectively removing them from circulation.
+4. The Location Assert Fee transaction is recorded on the blockchain.
+
+## Rationale
+
+The proposed change to IOT tokens as the method of payment for Location Assert Fees aligns with the rules established in HIP-51 and addresses the lack of incentive to pay for onboarding using HNT tokens. This change aims to drive value into the IOT ecosystem and provide IOT tokens with a small amount of utility, benefiting both the Helium and IOT communities.
+
+## Backwards Compatibility
```

Comment:
I'm confused by this section. What do you mean by backwards compatible here? It might be worth rephrasing this or adding this into an Deployment Impact section.